src/main.cpp

```diff
@@ -1606,8 +1606,9 @@ void MainFrame::OnTogBtnOnOff(wxCommandEvent& event)
         }
         else
         {
-            char temp[1024];
-            strncpy(temp, wxGetApp().m_psk_callsign.ToUTF8(), wxGetApp().m_psk_callsign.Length());
+            char temp[9];
+            memset(temp, 0, 10);
```

Comment:
@tmiw I think this is writing beyond the end of the array

Reply:
Yep, thanks! Updated.